### DISCUSSION TITLE
Some help with IKEA Tradfri Roller Blinds?

### DISCUSSION THREAD

**Discussion by aatifk** [Koenkk/zigbee2mqtt#17905] [Devices]
I'm in need of a little help, please. I have a total of four IKEA Tradfri roller blinds in my setup ([E1757](https://www.zigbee2mqtt.io/devices/E1757.html#ikea-e1757)) and I seem to be having two separate issues.

One is with how the blinds are paired. In my setup, all four of them are paired directly to the coordinator without using the IKEA Tradfri Repeater ([E1746](https://www.zigbee2mqtt.io/devices/E1746.html)). However, the device page for the Tradfri blind suggests using the signal repeater as part of the pairing process. I'm wondering if there's any benefit to doing it this way? Should I attempt to pair them again in this manner?

The second issue is with adding these blinds to a group. There are two in one room, and I created a group using the frontend with both blinds. I can't get this group to do anything in Home Assistant though. It shows up in HA but open/close doesn't do anything. Did I miss a step in creating the group? 

Finally, how do I get the open/close remote that comes with these blinds to work with them alongside remaining a part of Z2M and HA? 

Help much appreciated, please.

**Comment by sjorge** [comment]
> Finally, how do I get the open/close remote that comes with these blinds to work with them alongside remaining a part of Z2M and HA?

You can bind the remote to a blind, but all mine were already on a fw that does not support binding to groups, so you will have to bind them to a blind individually.

<img width="1905" alt="image" src="https://github.com/Koenkk/zigbee2mqtt/assets/379665/05125798-9c81-4aaf-a2d6-5ebacf4d680f">

You also need to unbind closure from the coordinator.

**Reply by aatifk** [reply]
oh thank you so much! I will try this right away. Question: can one remote be bound to two blinds?

**Reply by sjorge** [reply]
Yes, i think up to 10
